$NetBSD: patch-at,v 1.1 2016/07/03 17:50:31 dholland Exp $

NetBSD has POSIX signals since 2.0 at least.
Don't unpack archive, since it is hard to fix,
unpack before patch phase instead.

--- config/install.sh.orig	2007-06-12 20:43:58.000000000 +0400
+++ config/install.sh	2009-03-19 00:47:06.000000000 +0300
@@ -316,7 +316,7 @@
 if [ -x "$RUNDIR"/run.$ARCH-$OPSYS ]; then
     vsay $this: Run-time system already exists.
 else
-    "$CONFIGDIR"/unpack "$ROOT" runtime
+#    "$CONFIGDIR"/unpack "$ROOT" runtime
     cd "$BASEDIR"/runtime/objs
     echo $this: Compiling the run-time system.
     $MAKE -f mk.$ARCH-$OPSYS $EXTRA_DEFS
